aiida/cmdline/params/types/path.py

```diff
@@ -99,7 +99,7 @@ def convert(self, value, param, ctx):
     def checks_url(self, url, param, ctx):
         """Check whether URL is reachable within timeout."""
         try:
-            urllib.request.urlopen(url, timeout=self.timeout_seconds)
+            urllib.request.urlopen(url, timeout=self.timeout_seconds)  # pylint: disable=consider-using-with
```

Comment:
Also here, even though not crucial, might as well just use a context manager

Reply:
done